content/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,837 @@
+
+   ```
+   if (o_color_isBound) {
+      worldPosition.x += MaterialSrg::m_xOffset;
+      worldPosition.y += MaterialSrg::m_yOffset;
+   }
+   ```
+
+1. Repeat steps 2 and 3 with the depth pass in `VegetationBending_DepthPass.azsli`.
+
+1. Save both files and open your level in the **Editor** from the previous steps. 
+
+You should see that the tree entity is offset, but the shader ball is not! That's because the shader ball doesn't have a vertex color stream.
+
+{{< image-width src="/images/atom-guide/vegetation-bending-tutorial/optionoffset.png" width="100%" alt="The tree and the shader ball in the Editor, with only the tree offset from adding the shader option." >}}
+
+### Delete previous offset code
+Let's delete the previous offset code so that we can implement vegetation bending. 
+
+Delete the following: 
+* The code added to adjust the position in the vertex shaders of both the forward pass and depth pass.
+* The `m_xOffset` and `m_yOffset` variable declarations in the `MaterialSrg`, in the `VegetationBending_Common.azsli` file.
+* The `xOffset` and `yOffset` properties in the `VegetationBendingPropertyGroup.json` file. However,  keep the rest of the file and its connection in `VegetationBending.materialtype`.
+
+Make sure to keep the declarations for `m_optional_color` and `o_color_isBound`.
+
+### Create materials for the tree
+Let's add some textures to make our tree look more realistic! For the tree, we need 3 materials: for the trunk, the branches, and the leaves.
+
+1. Open the Editor, and then the **Material Editor**.
+
+1. Create a new material of the **VegetationBending** material type named `aspen_leaf.material`, and save it in the same folder you saved your previous material in, such as the `Materials` folder.
+
+1. Set the base color texture of the material. In the **Inspector**, under the **Base Color** > **Texture** property, click {{< icon "file-folder.svg" >}} and choose `aspen_leaf_basecolora.tif`.
+
+   {{< note >}}
+   The suffix `_basecolora` tells the engine to process the texture with a specific [*preset*](/docs/user-guide/assets/texture-settings/texture-presets). Appending a suffix to the name of a texture tells the engine to use the corresponding preset. In this case, we are using the `_basecolora` preset because this texture has the base color in the rgb channels and the opacity in the alpha channel.
+   {{< /note >}}
+
+1. Set the opacity mode. For the **Opacity** > **Opacity Mode** property, select `Cutout`. We need to set this to `Cutout` because the leaf texture has transparent parts.
+
+1. Under **General Settings**, enable **Double-sided**. This renders both sides of the material.
+
+1. Save your leaf material. Repeat with the branch and the trunk, but instead of making whole new materials, make the leaf material the parent of the branch and trunk materials so the properties stay constant for all 3.
```

Comment:
- Split into two steps. 
- "Repeat" step was getting long, so I suggest splitting the sentences.
- Added other terms to avoid ambiguity. 
```suggestion
1. Save your `aspen_leaf.material` material. 
2. Repeat the above steps for the branch and the trunk materials, but don't make new materials. Instead, make the branch and trunk materials be children of the leaf material. This allows the material properties to stay constant for all three.
```